trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js

```diff
@@ -171,7 +171,7 @@ const MOCK_TFM_DEAL_AIN_SUBMITTED = {
     parties: {
       agent: { partyUrn: '', partyUrnRequired: false },
       buyer: { partyUrn: '', partyUrnRequired: false },
-      exporter: { partyUrn: '00307249', partyUrnRequired: true },
+      exporter: { partyUrn: '"00307249"', partyUrnRequired: true },
```

Comment:
Can this please be single quotes, to keep it consistent.

Reply:
Ah good catch! Forgot to undo that one